lib/tasks/foreman_plugin_template_tasks.rake

```diff
@@ -13,11 +13,10 @@ end
 # Tests
 namespace :test do
   desc 'Test ForemanPluginTemplate'
-  Rake::TestTask.new(:foreman_plugin_template) do |t|
-    test_dir = File.expand_path('../../test', __dir__)
-    t.libs << 'test'
-    t.libs << test_dir
-    t.pattern = "#{test_dir}/**/*_test.rb"
+  Rake::TestTask.new(:foreman_plugin_template => 'db:test:prepare') do |t|
+    t.libs << ForemanPluginTemplate::Engine.root.join('test')
+    t.pattern = ForemanPluginTemplate::Engine.root.join('test', '**', '*_test.rb')
+    t.test_files = [Rails.root.join('test/unit/foreman/access_permissions_test.rb')]
```

Comment:
IMHO this is the important line of this PR and we should get that merged.

Reply:
I also think that this should get merged because we just stumbled across some permissions issue during plugin development and I realized only now that this permissions test got lost during the move to the unified github actions workflow.